bob-apps/bin/bobd.rs

```diff
@@ -120,6 +148,115 @@ async fn main() {
     }
 }
 
+fn configure_testmode(sub_matches: &ArgMatches) -> Result<(ClusterConfig, NodeConfig), String> {
+    let mut addresses = Vec::with_capacity(1);
+    let port = match sub_matches.value_of("grpc-port") {
+        Some(v) => v.parse().unwrap(),
```

Comment:
`unwrap` will panic. You should add `anyhow` context and return Error with `?` operator

Reply:
Done: [1e08373](https://github.com/qoollo/bob/pull/588/commits/1e0837312a5aa82e7351366ebf65dff582dd9a9c)